downstream/modules/devtools/proc-self-service-download-tar.adoc

```diff
@@ -18,15 +18,15 @@ $ export DYNAMIC_PLUGIN_ROOT_DIR=/path/to/<automation-portal-plugins-local-dir-c
 . In a browser, navigate to the
 link:{PlatformDownloadUrl}[Red Hat Ansible Automation Platform Product Software downloads page].
 and select the *Product Software* tab.
-. Click *Download now* next to the *Ansible plug-ins for Red Hat Developer Hub Setup Bundle* to download the latest `.tar` file for the plug-ins.
+. Click *Download now* next to *Ansible self-service automation portal* to download the latest `.tar` file for the plug-ins.
```

Comment:
Click *Download now* next to *Ansible self-service automation portal Setup Bundle* to download the latest version of the plug-ins.